src/Parser/DeathKnight/Blood/Modules/Talents/Bonestorm.js

```diff
@@ -61,9 +63,9 @@ class Bonestorm extends Analyzer {
   suggestions(when) {
     when(this.suggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<React.Fragment>Try to cast <SpellLink id={SPELLS.BONESTORM_TALENT.id} /> only if you can reliable hit 2 or more targets to maximize the damage and healing. Casting <SpellLink id={SPELLS.BONESTORM_TALENT.id} /> with only one target in range is a DPS and HPS loss, use <SpellLink id={SPELLS.DEATH_STRIKE.id} /> instead.</React.Fragment>)
+        return suggest(<React.Fragment>Try to cast <SpellLink id={SPELLS.BONESTORM_TALENT.id} /> only if you can reliable hit 2 or more targets to maximize the damage and healing. Casting <SpellLink id={SPELLS.BONESTORM_TALENT.id} /> with only one target in range is only a minor DPS gain (~10 DPS) at the cost of pooling Runic Power, use <SpellLink id={SPELLS.DEATH_STRIKE.id} /> instead.</React.Fragment>)
           .icon(SPELLS.BONESTORM_TALENT.icon)
-          .actual(`${ formatPercentage(actual) }% casts hit 2 or more targets`)
+          .actual(`${ formatPercentage(actual) }% casts hit ${SUGGESTED_MIN_TARGETS_FOR_BONESTORM} or more targets`)
           .recommended(`${ formatPercentage(recommended) }%is recommended`);
```

Comment:
You're going to suggest hitting 1.5 or more targets?

Reply:
Yes, casting Bonestorm is ST is only a ~10DPS increase while you're fairly vulnerable while pooling 100RP. Casting it with 2 targets is a ~300DPS incease and worth is using for purely DPS. So kinda went with something inbetween to not suggest people to cast BS on a regular basis in ST while not being to hard on people that want to use it purely for damage.

The break even-point used to be 2 in Legion for pure damage but AP modifiers changed.

Reply:
10 dps increase sounds awefully margin of error, but I'll take your word for it 😄